handlers/tealium.js

```diff
@@ -66,3 +66,56 @@ export const handler = async (lambdaEvent) => {
     return Promise.resolve('Nothing processed')
   }
 }
+
+export const bqHandler = async (rows) => {
+  if (typeof rows !== "undefined") {
+    const validEvents = [];
+    rows.forEach((row) => {
+      try {
+        validEvents.push(row);
+      } catch (error) {
+        if (process.env.LOG_LEVEL === "debug") {
+          rollbar.error("Event.fromRecord(row) error", error, {
+            row: row,
+          });
+        }
+
+        if (process.env.LOG_LEVEL === "debug") {
+          console.log("BigQuery Event Error: ", error.toString());
+        }
+      }
+    });
+  
+    if (validEvents.length > 0) {
```

Comment:
Instead of `validEvents` (since it's a big query row, not a snowplow event), just use rows.
```suggestion
    if (rows.length > 0) {
```